plugins/CoreVisualizations/CoreVisualizations.php

```diff
@@ -37,6 +39,21 @@ public function deleteUser($userLogin)
         ViewDataTableManager::clearUserViewDataTableParameters($userLogin);
     }
 
+    public function addViewDataTable(&$viewDataTable)
+    {
+        if (Common::getRequestVar('pivotBy', '')) {
+            $tableToRemove = 'Visualizations\HtmlTable';
+        } else {
+            $tableToRemove = 'HtmlTable\PivotBy';
```

Comment:
I don't really understand why we need to remove some viewDataTable anyway, could you add a comment?